.circleci/config.yml

```diff
@@ -1,44 +1,25 @@
 version: 2
-jobs:
-  build:
-    working_directory: ~/mimiker
-    docker:
-      - image: rafalcieslak/mimiker-build-base:1.1
-    steps:
-      - run:
-          name: 'Trigger basic jobs'
-          command: |
-            # One day this function will become a CircleCI built-in.
-            function trigger_job() {
-              curl --user ${CIRCLE_API_TOKEN}: \
-                --data build_parameters[CIRCLE_JOB]=$1 \
-                --data revision=$CIRCLE_SHA1 \
-                https://circleci.com/api/v1.1/project/github/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/tree/$CIRCLE_BRANCH
-            }          
-            trigger_job verify-formatting
-            trigger_job verify-pep8
-            trigger_job compile
 
-  verify-formatting:
-    working_directory: ~/mimiker
-    docker:
-      - image: rafalcieslak/mimiker-build-base:1.1
+defaults: &defaults
```

Comment:
`defaults:` creates a set of arbitrary values to be applied later. `&XYZ` specifies that the name of this defaults set shall be `XYZ`.